src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-bucket-list/rgw-bucket-list.component.html

```diff
@@ -22,3 +22,25 @@
                          [selection]="expandedRow">
   </cd-rgw-bucket-details>
 </cd-table>
+
+<ng-template #bucketSizeTpl
+             let-row="row">
+  <cd-usage-bar *ngIf="(row.bucket_quota.max_size > 0); else noSizeQuota"
+             [totalBytes]="1"
+             [usedBytes]="row.size_usage">
+  </cd-usage-bar>
+<ng-template #noSizeQuota>
```

Comment:
Indentation is missing here.
How about naming the template `sizeUsage`?

BTW is this template actually used?

Reply:
The progress bar has to be shown only when quota limit exists (for size : max_size>0 & 
object: max_objects>0) and if no quota exists then just the data is displayed without the progress bar, so accordingly used the name noSize/ObjectQuota